apps/design/backend/src/worker/worker.ts

```diff
@@ -1,45 +1,46 @@
 import { extractErrorMessage, sleep } from '@votingworks/basics';
 
-import { BackgroundTask, Store } from '../store';
+import { Workspace } from '../workspace';
+import { processBackgroundTask } from './tasks';
 
-async function processBackgroundTask(
-  _store: Store, // eslint-disable-line @typescript-eslint/no-unused-vars
-  _task: BackgroundTask // eslint-disable-line @typescript-eslint/no-unused-vars
-): Promise<void> {
-  // Simulate a long-running operation
-  // TODO(arsalan): Implement actual processing logic
-  await sleep(3000);
-}
+export async function processNextBackgroundTaskIfAny({
+  log = console.log, // eslint-disable-line no-console
```

Comment:
Tests can override with this with a `jest.fn` to avoid cluttering test output

Reply:
Alternatively just use `console.log` and [`suppressingConsoleOutput`](https://github.com/votingworks/vxsuite/blob/5fc65180bf45e1a4617095f3c7925531c82764e0/libs/test-utils/src/console.ts#L97-L101). No strong preference.

Reply:
Ah good reminder! Decided to use `suppressingConsoleOutput` so as not to introduce a second pattern for that: https://github.com/votingworks/vxsuite/pull/4462/commits/a366f85e00c50ae53442dce8f39237f5acbb89b8